configuration/items.md

```diff
@@ -310,17 +313,67 @@ The following guidelines apply to user-added icon files:
 Please check the user interface documentation if in doubt.
 Note that image files with the wrong file ending will be ignored.
 
 Users may substitute their own icon for an icon from the default icon set by placing a file in the `$OPENHAB_CONF/icons/classic/` folder with the same filename as the name of the icon being substituted.
 
+#### Icon sources
+
+Some user interfaces supports other icons than those provided by openHAB.
+Thus, the most generic reference to an icon is composed of 3 segments separated by a colon:
+
+1. The first segment is the icon source, e.g. "oh"
+
+1. The second segment is the icon set from this source, e.g. "classic"
+
+1. The third segment is the name of an icon in this set, e.g. "switch"
+
+If the value contains only two segments, the first segment is assumed to be the icon source and the second the icon name.
+In this case, the icon set is to "classic" by default.
+This is used in particular when the icon source only provides a single set of icons.
+As an example, "&lt;material:favorite&gt;" references the "favorite" icon from the Material icons.
+"&lt;oh:switch&gt;" references the "switch" icon from the openHAB classic icon set.
+
+If the value contains only one segment, it is assumed to be the name of an icon from the openHAB classic icon set.
+As an example, "&lt;switch&gt;" references the "switch" icon from the openHAB classic icon set.
+
+Here are few examples showing the different options:
+
+```java
+Switch Livingroom_Light "Livingroom Ceiling Light" <switch>
+Switch Parent_Bedroom_Light "Parent Bedroom Light" <oh:switch>
+Switch Child_Bedroom_Light "Child Bedroom Light" <oh:classic:switch>
+Switch Bathroom_Light "Bathroom Light" <material:lightbulb>
+Switch Kitchen_Light "Kitchen Light" <f7:lightbulb>
+Switch Garage_Light "Garage Light" <if:mdi:lightbulb>
+```
+
+Here is the list of available icon sources and how they are supported by the major user interfaces.
+
+| Source name          | Source description                    | Main UI    | Basic UI                          | Android app       | iOS app           |
+| -------------------- | ------------------------------------- | ---------- | --------------------------------- | ----------------- | ----------------- |
+| `oh`                | Icons provided via the openHAB server | Supported  | Supported                         | Supported         | Supported         |
+| `material`         | Material icons                        | Supported  | Supported                         | Not yet supported | Not yet supported |
+| `f7`                | Framework7 icons                      | Supported  | Not yet supported                 | Not yet supported | Not yet supported |
+| `if` or `iconify` | iconify icons                         | Supported  | Supported but needs to be enabled | Not yet supported | Not yet supported |
+
+You can find a list of [all Material icons here](https://fonts.google.com/icons?icon.set=Material+Icons).
+
```

Comment:
You could move these links to a new coloumn in the table above. For source "oh" there's already a docs page that can be linked.

Reply:
Or even by adding the link in the source description column.

Reply:
Done